File: docs/fundamentals/runtime-libraries/snippets/System/Char/Overview/csharp/textelements2.cs
// <Snippet3>
using System;

public class Example5
{
    public static void Main()
    {
        string result = String.Empty;
        for (int ctr = 0x10107; ctr <= 0x10110; ctr++)  // Range of Aegean numbers.
            result += Char.ConvertFromUtf32(ctr);

        Console.WriteLine($"The string contains {result.Length} characters.");
    }
}
// The example displays the following output:
//     The string contains 20 characters.
// </Snippet3>

File: docs/fundamentals/runtime-libraries/snippets/System/Char/Overview/vb/textelements2.vb
' Visual Basic .NET Document
Option Strict On

' <Snippet3>
Module Example5
    Public Sub Main()
        Dim result As String = String.Empty
        For ctr As Integer = &H10107 To &H10110     ' Range of Aegean numbers.
            result += Char.ConvertFromUtf32(ctr)
        Next
        Console.WriteLine("The string contains {0} characters.", result.Length)
    End Sub
End Module
' The example displays the following output:
'     The string contains 20 characters.
' </Snippet3>
